src/plugins/openmeteo/datasource.py

```
@@ -0,0 +1,333 @@
+                    logger.warning(f"No hourly data in response for {sensor.location.name}")
+                    current_start = chunk_end + timedelta(days=1)
+                    continue
+                    
+                hourly_data = data['hourly']
+                times = hourly_data.get('time', [])
+                values = hourly_data.get(sensor.metadata['parameter_code'], [])
+                
+                measurements = []
+                measurement_count = 0
+                
+                for i, (time_str, value) in enumerate(zip(times, values)):
```

Ruff (0.12.2)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)